server/src/main/java/org/opensearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -782,6 +804,16 @@ BlobContainer getBlobContainer() {
         return blobContainer.get();
     }
 
+    // package private, only use for testing
+    BlobContainer getRootBlobContainer() {
+        return rootBlobContainer.get();
+    }
```

Comment:
For my understanding how is this different from `blobContainer`

Reply:
`blobContainer` in this class points to the `basePath`.